packages/venia-ui/lib/i18n/en_US.json

```diff
@@ -37,5 +37,7 @@
     "accountMenu.addressBookLink": "Address Book",
     "accountMenu.savedPaymentsLink": "Saved Payments",
     "accountMenu.communicationsLink": "Communications",
-    "accountMenu.accountInfoLink": "Account Information"
+    "accountMenu.accountInfoLink": "Account Information",
+    "That page could not be found. Please try again.": "That page could not be found. Please try again.",
+    "Something went wrong. Please try again.": "Something went wrong. Please try again."
```

Comment:
Ah, yes, here I would use id format like the above `component.thing` rather than a string.

Reply:
Done,
Thank you

Reply:
Thanks for updating it - seems like we're still going to be figuring this out for a while, but I don't think you need to change it again in your PR.